weather_dl_v2/cli/app/cli_config.py

```diff
@@ -12,11 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
+import pkg_resources
```

Comment:
Lets maintain lexicographical ordering

Reply:
resolved in [6cf31cf](https://github.com/google/weather-tools/pull/422/commits/6cf31cf2866762730543a057927df3c1283ab1a0)